website/docs/addon_slack.md

```diff
@@ -15,83 +15,130 @@ import Tabs from '@theme/Tabs';
-could be configured, `Families`, `Task names` (regex available), `Host names`, `Product names` (regex available) combination is needed.
+![](assets/slack/auth_key.png)
 
-Eg. If I want to be notified when render is published from Maya, setting is:
+For the Slack addon to function properly, it's essential to enter the authentication token:
 
-- family: 'render'
-- host: 'Maya'
+- For all projects:
+  - `ayon+settings://slack/token`
+- For a specific project, replace {PROJECT_NAME} with your project's name:
+  - `ayon+settings://slack/token?project={PROJECT_NAME}`
 
-### Messages to channels
+You'll find this token in the Slack dashboard once the app is installed. Remember, you can create separate tokens for use with different projects.
 
-#### Channels
-Multiple messages could be delivered to one or multiple channels, by default app allows Slack bot
-to send messages to 'public' channels (eg. bot doesn't need to join the channel first).
+![Get token](assets/slack/slack_token.png)
 
-![Configure module](assets/slack_project.png)
+### Profiles
 
-#### Upload thumbnail
-Integration can upload 'thumbnail' file (if present in an instance), for that bot must be 
-manually added to each target channel by Slack admin first!
-(In target channel write: ```/invite @ayonNotifier``)
+![](assets/slack/profiles.png)
 
-#### Upload review
-Integration can upload 'review' file (if present in an instance), for that bot must be 
-manually added to each target channel by Slack admin first!
-(In target channel write: ```/invite @ayonNotifier``)
+Profiles determine when notifications are triggered. You can configure one or several profiles based on `Product types`, `Hosts`, `Task types`, and `Task names` or `Product names` (with regex support).
 
-This option is limited by `Upload review maximum file size` value in `Profiles` section. It might make sense to limit uploading
-of reviews only up to certain size. If the review file hits that limit, only link to studio accessible location will be inserted instead.
+For example, to receive notifications when a render is published from Maya, the settings would be:
 
-Burnin version of the review (usually .mp4) is preferred if present.
+- Product type: 'render'
+- Host: 'Maya'
 
+### Messages to channels
+
+![Configure module](assets/slack/messages_to_channels.png)
+
+- **Channels:**
+Specify one or more channels where you'd like the plugin to send messages.
+:::info
+Please note that the Slack bot can access public channels by default, so there's no need for it to join them first.
+:::
+- **Upload thumbnail** & **Upload review:**
+The plugin is capable of uploading a 'thumbnail' or 'review' from an instance (if they are present in an instance). However, your Slack bot must be manually invited to each target channel by a Slack admin.
+:::tip
+To add your bot to a target channel, simply mention it in the message field with the command: `/invite @ayonNotifier`.
+:::
```

Comment:
The info message here and the tip here seem to be very related to the same thing.

I wonder if it makes more sense to combine them, by writing e.g.

```suggestion

- **Upload thumbnail** & **Upload review:**
The plugin is capable of uploading a 'thumbnail' or 'review' from an instance (if they are present in an instance). However, your Slack bot must be manually invited to each target channel by a Slack admin if they are private channels.
:::note
Please note that the Slack bot can access public channels by default, so there's no need for it to join them first.
To add your bot to a private target channel, simply mention it in the message field with the command: `/invite @ayonNotifier` inside that channel.
:::
```

Reply:
Let me know if this made it better.
https://github.com/ynput/ayon-documentation/pull/218/commits/b91c4749cfb3e59fddc3a7a933310f161ebfda47